storage/nfsrahead/pkg.yaml

```diff
@@ -29,28 +29,29 @@ steps:
         export CFLAGS="${CFLAGS} -I/usr/local/include"
 
         ./configure \
-          --disable-gss \
-          --disable-ipv6 \
-          --disable-mount \
-          --disable-nfsdcld \
-          --disable-nfsdctl \
-          --disable-nfsv4 \
-          --disable-nfsv41 \
-          --disable-uuid \
-          --enable-junction=no \
-          --with-nfsconfig=/usr/local/etc/nfsrahead/nfs.conf \
-          --with-rpcgen=internal \
-          --prefix=/usr/local
+            --disable-gss \
+            --disable-ipv6 \
+            --disable-mount \
+            --disable-nfsdcld \
+            --disable-nfsdctl \
+            --disable-nfsv4 \
+            --disable-nfsv41 \
+            --disable-uuid \
+            --enable-junction=no \
+            --with-nfsconfig=/usr/local/etc/nfsrahead/nfs.conf \
+            --with-rpcgen=internal \
+            --prefix=/usr/local
     build:
       - |
+        make clean
```

Comment:
I'm not sure how this would fix things, the `99` rules is in talos rootfs, not here, so a proper fix would be to either drop it from `pkgs` like as in https://github.com/siderolabs/pkgs/blob/main/systemd-udevd/pkg.yaml#L33 or patch it there

Reply:
The .rules is in the nfs-utils tar.gz. Because it's already generated, running a make on it won't recreate it. However, running a make clean does delete it and then uses the .in file to regenerate it correctly.

https://git.linux-nfs.org/?p=steved/nfs-utils.git;a=tree;f=tools/nfsrahead;h=d2184c6deb0c5fa9b6a97d0a731cbe4944149c37;hb=HEAD